js/front/dataset/resource-modal.vue

```diff
@@ -7,6 +7,8 @@
 
         <dl class="dl-horizontal dl-wide">
           <dt>{{ _('URL') }}</dt>
+          <dd><a :href="resource.contentUrl" @click="onClick">{{resource.contentUrl}}</a></dd>
+          <dt>{{ _('Permalink') }}</dt>
```

Comment:
To me a permalink is stable both in the URI *and* in the content delivered, see a quote from that wikipedia page:

> In the context of systems that support versioning, such as most wikis, a permalink is commonly understood as a link to a *specific version*.

So semantically, we should name the `url` permalink and the redirecting one `latest` 😉